learning/katas/go/Core Transforms/Flatten/Flatten/cmd/main.go

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import "github.com/apache/beam/sdks/go/pkg/beam"
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
```

Comment:
Please provide main() function to run the pipeline normally.
You can refer to other main.go files for reference.

Reply:
Ya, saw that after reading your email!  Figuring out the squash.

Reply:
@henryken -- I think I got it (can't believe I overlooked that)  :-)  let me know how it looks, thanks!

Reply:
Looks good now. Thanks!